packages/bot/src/components/transactions.jsx

```diff
@@ -0,0 +1,169 @@
+// import className from 'classnames';
+import { Money,Popover } from 'deriv-components';
+import { PropTypes }     from 'prop-types';
+import React             from 'react';
+import { Scrollbars }    from 'tt-react-custom-scrollbars';
+import {
+    BuyPriceIcon,
+    ExitSpotIcon,
+    EntrySpotIcon,
+    PendingIcon,
+    RefrenceIdIcon,
+    SettledIcon,
+}                        from './Icons.jsx';
+import IconTradeType     from './icon-trade-types.jsx';
+import { connect }       from '../stores/connect';
+import { translate }     from '../utils/tools';
+import '../assets/sass/transactions.scss';
+
+const Transaction = ({ contract }) => {
+    return (
+        <table className='transactions__item'>
+            <tbody>
+                <tr className='transactions__tr'>
```

Comment:
Just a nitpick, but imo we should use less verbose names for classes, e.g. `transactions__row` and `transactions__cell`.